### ISSUE TITLE
Fatal error: exception JBasics.Class_structure_error("Illegal method index (does not refer to a constant method)")

### ISSUE THREAD

**Issue by mcandre** [facebook/infer#264]
# Summary

I get a strange error in Infer when analyzing some proprietary code.

# Trace

```
$ infer -- ./gradlew clean build
Running and capturing gradle compilation...
14:21:26.603 [ERROR] [system.err] Note: Some input files use unchecked or unsafe operations.
14:21:26.604 [ERROR] [system.err] Note: Recompile with -Xlint:unchecked for details.
14:21:28.939 [ERROR] [system.err] objc[96190]: Class JavaLaunchHelper is implemented in both /Users/andrew.pennebaker/.jenv/candidates/java/1.8.0_65/bin/java and /Users/andrew.pennebaker/.jenv/candidates/java/1.8.0_65/jre/lib/libinstrument.dylib. One of the two will be used. Which one is undefined.
Warning: unexpected attribute found: Code
Warning: unexpected attribute found: Code
Warning: unexpected attribute found: Code
Warning: unexpected attribute found: Code
Warning: unexpected attribute found: Code
Warning: unexpected attribute found: Code
Warning: unexpected attribute found: Code
Warning: unexpected attribute found: Code
Warning: unexpected attribute found: Code
Warning: unexpected attribute found: Code
Fatal error: exception JBasics.Class_structure_error("Illegal method index (does not refer to a constant method)")
Traceback (most recent call last):
  File "/usr/local/bin/infer", line 180, in <module>
    main()
  File "/usr/local/bin/infer", line 145, in main
    capture_exitcode = imported_module.gen_instance(args, cmd).capture()
  File "/usr/local/Cellar/infer/0.6.0/libexec/infer/lib/python/inferlib/capture/gradle.py", line 86, in capture
    return util.run_compilation_commands(cmds, clean_cmd)
  File "/usr/local/Cellar/infer/0.6.0/libexec/infer/lib/python/inferlib/capture/util.py", line 45, in run_compilation_commands
    if cmd.start() != os.EX_OK:
  File "/usr/local/Cellar/infer/0.6.0/libexec/infer/lib/python/inferlib/analyze.py", line 615, in start
    self.run_infer_frontend()
  File "/usr/local/Cellar/infer/0.6.0/libexec/infer/lib/python/inferlib/analyze.py", line 361, in run_infer_frontend
    self.args.analyzer
  File "/usr/local/Cellar/infer/0.6.0/libexec/infer/lib/python/inferlib/analyze.py", line 239, in run_command
    raise e
subprocess.CalledProcessError: Command '[u'/usr/local/Cellar/infer/0.6.0/libexec/infer/lib/python/inferlib/../../../bin/InferJava', u'-classpath', u'/Users/andrew.pennebaker/src/sf-social-profiles-models/build/classes/main:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/org.json/json/20150729/7ad48f520e9f94787fdbf2beb9916c76971bdcb0/json-20150729.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/com.jayway.jsonpath/json-path/2.0.0/26b8555596b3fb9652c1ffe193fa9123945b32cc/json-path-2.0.0.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/javax.validation/validation-api/1.1.0.Final/8613ae82954779d518631e05daa73a6a954817d5/validation-api-1.1.0.Final.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/javax/javaee-api/7.0/51399f902cc27a808122edcbebfaa1ad989954ba/javaee-api-7.0.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/org.apache.commons/commons-lang3/3.4/5fe28b9518e58819180a43a850fbc0dd24b7c050/commons-lang3-3.4.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/org.apache.commons/commons-collections4/4.1/a4cf4688fe1c7e3a63aa636cc96d013af537768e/commons-collections4-4.1.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/com.googlecode.libphonenumber/libphonenumber/7.2.2/b6975e439e0b44984098d1e5eb9f422ce1431ecb/libphonenumber-7.2.2.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/com.googlecode.libphonenumber/geocoder/2.32/866edecb30ccce8dc2c7abe62e3449e413e752ed/geocoder-2.32.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/com.maxmind.geoip2/geoip2/2.3.0/81d1b0440f2d9086955f41dee79e0a9ed3d07da3/geoip2-2.3.0.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/org.geotools/gt-main/14.1/85a173a34b246b65acd81898221ba957db6d5fc7/gt-main-14.1.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/io.dropwizard/dropwizard-jackson/0.7.1/6bc655d16acdb556001b1d677fab6c69f7ba296a/dropwizard-jackson-0.7.1.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/com.netflix.astyanax/astyanax-entity-mapper/3.8.0/889bf1208f74a15d4f00a3e5c81c8f779e2628a3/astyanax-entity-mapper-3.8.0.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/com.netflix.astyanax/astyanax-cassandra/3.8.0/8b9704784b34229faace4aa986d08b80dba6aac6/astyanax-cassandra-3.8.0.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/org.slf4j/slf4j-api/1.7.10/5b338f206827d88acd890739f400a9b7664e5984/slf4j-api-1.7.10.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/net.minidev/json-smart/2.1.1/922d12fb1f394e2b6999ae0f7936ab13f4dffb81/json-smart-2.1.1.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/com.sun.mail/javax.mail/1.5.0/ec2410fdf7e0a3022e7c2a2e6241039d1abc1e98/javax.mail-1.5.0.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/com.googlecode.libphonenumber/prefixmapper/2.32/f9d0255421df96c38b4859d2da788f4f38c76df3/prefixmapper-2.32.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/com.maxmind.db/maxmind-db/1.0.0/b140295a52005aaf224b6c711ad4ecb38b1da155/maxmind-db-1.0.0.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/com.google.http-client/google-http-client/1.20.0/93d82db2bca534960253f43424b2ba9d7638b4d2/google-http-client-1.20.0.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.core/jackson-databind/2.5.3/c37875ff66127d93e5f672708cb2dcc14c8232ab/jackson-databind-2.5.3.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/org.geotools/gt-api/14.1/f53d724f66a6c29e07412a8d4c5d0333b4b01bfa/gt-api-14.1.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/com.vividsolutions/jts/1.13/3ccfb9b60f04d71add996a666ceb8902904fd805/jts-1.13.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/org.jdom/jdom/1.1.3/8bdfeb39fa929c35f5e4f0b02d34350db39a1efc/jdom-1.1.3.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/javax.media/jai_core/1.1.3/b179d2efb1174658483e8b41bf4ac9d2eb5de438/jai_core-1.1.3.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/com.google.guava/guava/17.0/9c6ef172e8de35fd8d4d8783e4821e57cdef7445/guava-17.0.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/io.dropwizard/dropwizard-util/0.7.1/49c3b2764b4f6ad1a97ae7148ac8a640488b0b29/dropwizard-util-0.7.1.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.datatype/jackson-datatype-guava/2.3.3/f3874cad8b324b8e1c4ac1501063f4bc3a790e96/jackson-datatype-guava-2.3.3.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.module/jackson-module-afterburner/2.3.3/9da53b15c4b708517208bbded61b9c4aa8824584/jackson-module-afterburner-2.3.3.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.datatype/jackson-datatype-joda/2.3.3/2592678ed4fa51dfcea0e52be99578581945c861/jackson-datatype-joda-2.3.3.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/ch.qos.logback/logback-classic/1.1.2/b316e9737eea25e9ddd6d88eaeee76878045c6b2/logback-classic-1.1.2.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/com.netflix.astyanax/astyanax-core/3.8.0/209a330c9856720b3ae8ccda8ccf819bc9cab362/astyanax-core-3.8.0.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/com.netflix.astyanax/astyanax-recipes/3.8.0/71cf76ec6ffd6142f3e73101b9d6a4c35a67cd2f/astyanax-recipes-3.8.0.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/javax.persistence/persistence-api/1.0/5725f57873e05e068803e2bf9d5a8ea3740ffec5/persistence-api-1.0.jar:/Users/andrew.pennebaker/.m2/repository/commons-lang/commons-lang/2.4/commons-lang-2.4.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/org.apache.cassandra/cassandra-all/2.0.12/84c12bd66a499c1fcc187aa889916671c55965b0/cassandra-all-2.0.12.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/org.xerial.snappy/snappy-java/1.0.5/10cb4550360a0ec6b80f09a5209d00b6058e82bf/snappy-java-1.0.5.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/org.codehaus.jackson/jackson-mapper-asl/1.9.2/95400a7922ce75383866eb72f6ef4a7897923945/jackson-mapper-asl-1.9.2.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/org.apache.cassandra/cassandra-thrift/2.0.12/470a72dafc8d16ccdb41342aeb8001b28dcd6e40/cassandra-thrift-2.0.12.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/org.apache.servicemix.bundles/org.apache.servicemix.bundles.commons-csv/1.0-r706900_3/ff0cd44f590a80c5c87aaa85a0d2bab2d350bc4a/org.apache.servicemix.bundles.commons-csv-1.0-r706900_3.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/org.codehaus.jettison/jettison/1.2/765a6181653f4b05c18c7a9e8f5c1f8269bf9b2/jettison-1.2.jar:/Users/andrew.pennebaker/.m2/repository/commons-codec/commons-codec/1.6/commons-codec-1.6.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/net.minidev/asm/1.0.2/63900a15f524db0b8c4fb2d9e24c0cb179842ea5/asm-1.0.2.jar:/Users/andrew.pennebaker/.m2/repository/javax/activation/activation/1.1/activation-1.1.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/org.geotools/gt-referencing/14.1/7154abcd8daacdd49c9067f5f131dd3e5a281c35/gt-referencing-14.1.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/joda-time/joda-time/2.3/56498efd17752898cfcc3868c1b6211a07b12b8f/joda-time-2.3.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/ch.qos.logback/logback-core/1.1.2/2d23694879c2c12f125dac5076bdfd5d771cc4cb/logback-core-1.1.2.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/com.github.stephenc.high-scale-lib/high-scale-lib/1.1.2/e3fbb2a95269eb8049bc72a3b0ce3208378a30f5/high-scale-lib-1.1.2.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/com.eaio.uuid/uuid/3.2/77ba5105d949cd589aff75400d9f7d3676691a46/uuid-3.2.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/net.jpountz.lz4/lz4/1.2.0/cd301e9a481eae5b68c53267100aeca43b782b9f/lz4-1.2.0.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/com.ning/compress-lzf/0.8.4/994da6bddb79a40c368d3040f2aa06b94faf6b1f/compress-lzf-0.8.4.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/commons-cli/commons-cli/1.1/11c98b99ad538f2f67633afd4d7f4d98ecfbb408/commons-cli-1.1.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/com.googlecode.concurrentlinkedhashmap/concurrentlinkedhashmap-lru/1.3/beb907bae0604fdc153cbcc2f0dc84d3ae35bf36/concurrentlinkedhashmap-lru-1.3.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/org.antlr/antlr/3.2/6b0acabea7bb3da058200a77178057e47e25cb69/antlr-3.2.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/org.codehaus.jackson/jackson-core-asl/1.9.2/8493982bba1727106d767034bd0d8e77bc1931a9/jackson-core-asl-1.9.2.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/jline/jline/1.0/e8f2d4690059d42a8f0f29195884a72a82ce4bb7/jline-1.0.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/com.googlecode.json-simple/json-simple/1.1/5e303a03d04e6788dddfa3655272580ae0fc13bb/json-simple-1.1.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/edu.stanford.ppl/snaptree/0.1/88caaa75fba09a4944f696c398c09da956f7c2c2/snaptree-0.1.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/org.mindrot/jbcrypt/0.3m/fe2d9c5f23767d681a7e38fc8986b812400ec583/jbcrypt-0.3m.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/com.yammer.metrics/metrics-core/2.2.0/f82c035cfa786d3cbec362c38c22a5f5b1bc8724/metrics-core-2.2.0.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/com.addthis.metrics/reporter-config/2.1.0/1f4c83a733634be5f550ed94ceb9475b17ce8ba6/reporter-config-2.1.0.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/com.thinkaurelius.thrift/thrift-server/0.3.7/e8182774da1b1dde3704f450837c79997b5d7025/thrift-server-0.3.7.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/net.sf.supercsv/super-csv/2.1.0/c6466dd0e28c034272b9f70a3f1896c03f1f2b27/super-csv-2.1.0.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/org.apache.thrift/libthrift/0.9.1/16c9cccf08caa385b5fc93934cb3216fe6ac6a72/libthrift-0.9.1.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/com.github.stephenc/jamm/0.2.5/422d3543c01df2f1d8bd1f3064adb54fb9e93f3/jamm-0.2.5.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/io.netty/netty/3.6.6.Final/e4e40738ce9bee0a92389cb739c94d7839778647/netty-3.6.6.Final.jar:/Users/andrew.pennebaker/.m2/repository/stax/stax-api/1.0.1/stax-api-1.0.1.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/asm/asm/3.3.1/1d5f20b4ea675e6fab6ab79f1cd60ec268ddc015/asm-3.3.1.jar:/Users/andrew.pennebaker/.m2/repository/commons-logging/commons-logging/1.1.1/commons-logging-1.1.1.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/com.googlecode.efficient-java-matrix-library/core/0.26/2df89c44ad92900f025a61e228bbbcd6abca3efd/core-0.26.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/commons-pool/commons-pool/1.5.4/75b6e20c596ed2945a259cea26d7fadd298398e6/commons-pool-1.5.4.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/org.geotools/gt-metadata/14.1/72135147a1af9b16d0bac34d02ce4be009d0441e/gt-metadata-14.1.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/jgridshift/jgridshift/1.0/82c163e82b1548c93a9e866dcfd24c66c590a829/jgridshift-1.0.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/net.sf.geographiclib/GeographicLib-Java/1.44/7e18468fafefedd5d1934ea830c1c9683ffd0632/GeographicLib-Java-1.44.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/org.antlr/antlr-runtime/3.2/31c746001016c6226bd7356c9f87a6a084ce3715/antlr-runtime-3.2.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/org.hibernate/hibernate-validator/4.3.0.Final/f2d0447bcdb27a48398215a33eb351b8a594e3a4/hibernate-validator-4.3.0.Final.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/com.lmax/disruptor/3.0.1/d68c363b12827c644bd60469827b862cad7dc0a2/disruptor-3.0.1.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/junit/junit/4.8.1/f2975548f836416306ef1dee748d956f04733915/junit-4.8.1.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/org.geotools/gt-opengis/14.1/47c8b6820c9b4f7652d1409e02abfa6b34e763db/gt-opengis-14.1.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/org.antlr/stringtemplate/3.2/6fe2e3bb57daebd1555494818909f9664376dd6c/stringtemplate-3.2.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/org.jboss.logging/jboss-logging/3.1.0.CR2/28725380c07f917ace4e511db21cc45e9ae5a72b/jboss-logging-3.1.0.CR2.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/net.java.dev.jsr-275/jsr-275/1.0-beta-2/7bdd90125e67cd38bee363208e32993c0c75e9be/jsr-275-1.0-beta-2.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/antlr/antlr/2.7.7/83cd2cd674a217ade95a4bb83a8a14f351f48bd0/antlr-2.7.7.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/com.google.code.findbugs/jsr305/2.0.3/5871fb60dc68d67da54a663c3fd636a10a532948/jsr305-2.0.3.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/org.yaml/snakeyaml/1.12/ebe66a6b88caab31d7a19571ad23656377523545/snakeyaml-1.12.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/org.apache.httpcomponents/httpclient/4.2.5/666e26e76f2e87d84e4f16acb546481ae1b8e9a6/httpclient-4.2.5.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/org.apache.httpcomponents/httpcore/4.2.4/3b7f38df6de5dd8b500e602ae8c2dd5ee446f883/httpcore-4.2.4.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.core/jackson-annotations/2.5.0/a2a55a3375bc1cef830ca426d68d2ea22961190e/jackson-annotations-2.5.0.jar:/Users/andrew.pennebaker/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.core/jackson-core/2.5.3/a8b8a6dfc8a17890e4c7ff8aed810763d265b68b/jackson-core-2.5.3.jar', u'-class_source_map', '/var/folders/g6/kpqw5r_j50b6c48v238rmm6400018s/T/classSourceMap_5fv6V6.json', u'-project_root', '/Users/andrew.pennebaker/src/sf-social-profiles-models', u'-results_dir', u'/Users/andrew.pennebaker/src/sf-social-profiles-models/infer-out', u'-verbose_out', '/var/folders/g6/kpqw5r_j50b6c48v238rmm6400018s/T/javac_6eBwhe.out', u'-models', u'/usr/local/Cellar/infer/0.6.0/libexec/infer/lib/python/inferlib/../../../lib/java/models.jar', u'-no-static_final']' returned non-zero exit status 2
```

# System

```
$ specs infer gradle java brew os
specs --version
0.21

infer --version
Infer version v0.6.0
Copyright 2009 - present Facebook. All Rights Reserved.

gradle --version

------------------------------------------------------------
Gradle 2.7
------------------------------------------------------------

Build time:   2015-09-14 07:26:16 UTC
Build number: none
Revision:     c41505168da69fb0650f4e31c9e01b50ffc97893

Groovy:       2.3.10
Ant:          Apache Ant(TM) version 1.9.3 compiled on December 23 2013
JVM:          1.8.0_65 (Oracle Corporation 25.65-b01)
OS:           Mac OS X 10.11.2 x86_64


mvn --version
Apache Maven 3.3.3 (7994120775791599e205a5524ec3e0dfe41d4a06; 2015-04-22T06:57:37-05:00)
Maven home: /Users/andrew.pennebaker/.jenv/candidates/maven/current
Java version: 1.8.0_65, vendor: Oracle Corporation
Java home: /Users/andrew.pennebaker/.jenv/candidates/java/1.8.0_65/jre
Default locale: en_US, platform encoding: UTF-8
OS name: "mac os x", version: "10.11.2", arch: "x86_64", family: "mac"

echo $CLASSPATH


echo $JAVA_HOME
/Users/andrew.pennebaker/.jenv/candidates/java/current

javac -version
javac 1.8.0_65

java -version
java version "1.8.0_65"
Java(TM) SE Runtime Environment (build 1.8.0_65-b17)
Java HotSpot(TM) 64-Bit Server VM (build 25.65-b01, mixed mode)

brew --version
Homebrew 0.9.5 (git revision 599b; last commit 2016-02-01)

system_profiler SPSoftwareDataType | grep 'System Version'
    System Software Overview:
      System Version: OS X 10.11.2 (15C50)
      System Integrity Protection: Enabled
```

**Comment by sblackshear** [comment]
This comes from the frontend issues with parsing Java 8 bytecode.

**Comment by jvillard** [comment]
This should be fixed in 0.8.0: the analysis of files containing Java 8 code should now be skipped instead of crashing the analysis. I've opened a separate issue to properly analyse Java 8 in Infer (#314).
